src/irr/ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -686,7 +693,7 @@ core::smart_refctd_ptr<asset::ICPUPipelineLayout> CMitsubaLoader::createPipeline
 	pcrng.size = sizeof(uint32_t);//instance data offset
 	pcrng.stageFlags = static_cast<asset::ISpecializedShader::E_SHADER_STAGE>(asset::ISpecializedShader::ESS_FRAGMENT | asset::ISpecializedShader::ESS_VERTEX);
 
-	core::smart_refctd_ptr<ICPUDescriptorSetLayout> ds0layout;
+	core::smart_refctd_ptr<ICPUDescriptorSetLayout> ds0layout; // needs to builtin and cached statically
```

Comment:
cant because of VT

Reply:
yeah, forgot to remove the comment... you can remove it, I guess.